new/cookbook-segment-deprecation.md

```diff
@@ -0,0 +1,51 @@
+---
+RFC: unassigned
+Author: Lamont Granquist <lamont@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Cookbook Segment Deprecation
+
+## Motivation
+
+    As a Chef User/Developer,
+    I want to be able to extend the structure of cookbooks without fighting with a wire protocol
+    written 6 years ago,
```

Comment:
Yeah, I find writing motivation descriptions completely demotivating and useless...